app/Http/Controllers/Three/SignupsController.php

```diff
@@ -70,4 +71,21 @@ public function destroy(Signup $signup)
 
         return $this->respond('Signup deleted.', 200);
     }
+
+    /**
+     * Updates a specific signup.
+     * PATCH /signups/:id
+     *
+     * @param SignupRequest $request
+     * @param \Rogue\Models\Signup $signup
+     * @return \Illuminate\Http\Response
+     */
+    public function update(SignupRequest $request, Signup $signup)
```

Comment:
Let's move this up so it goes `index`, `show`, `edit`, `update`, `destroy` & all the controllers are consistent!

Reply:
yes!